py/tests/scale.py

```diff
@@ -4,49 +4,74 @@
 import time
 import random
 
+def plane_commit(card, crtc, plane, fb, x, y, w, h) :
+	req = pykms.AtomicReq(card)
+	req.add(plane, {"FB_ID": fb.id,
+		"CRTC_ID": crtc.id,
+		"SRC_X": 0 << 16,
+		"SRC_Y": 0 << 16,
+		"SRC_W": fb.width << 16,
+		"SRC_H": fb.height << 16,
+		"CRTC_X": x,
+		"CRTC_Y": y,
+		"CRTC_W": w,
+		"CRTC_H": h,
+		"zpos": 0})
+	r = req.commit_sync()
+
 card = pykms.Card()
 res = pykms.ResourceManager(card)
-conn = res.reserve_connector("hdmi")
+#conn = res.reserve_connector("hdmi")
+conn = card.get_first_connected_connector
```

Comment:
Always using HDMI is bad, but using the first connector is not very good either. We should get the (optional) connector parameter from the cmdline and use that. Resman does get the first connected connector if the given parameter is empty.

And in any case, don't leave the old code behind a comment, as you do here an in other places and files.